pkg/controller/nodemaintenance/nodemaintenance_controller.go

```diff
@@ -19,12 +19,11 @@ import (
 	"k8s.io/client-go/tools/cache"
 	"k8s.io/klog"
 	"k8s.io/kubectl/pkg/drain"
-	kubevirtv1alpha1 "kubevirt.io/node-maintenance-operator/pkg/apis/kubevirt/v1alpha1"
+	nodemaintenanceapi "kubevirt.io/node-maintenance-operator/pkg/apis/kubevirt/v1beta1"
```

Comment:
Sorry for being unclear, I meant that this change should be made everywhere. Not just in the specific place I mentioned it.

Reply:
done. Sorry for the oversight.